pm.mirko.Atoms.yml

```diff
@@ -0,0 +1,219 @@
+    enable-if: active-gl-driver
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.toolchain-i386
+  - org.freedesktop.Sdk.Extension.golang
+
+cleanup:
+- "/include"
+- "/lib/pkgconfig"
+- "/man"
+- "/share/doc"
+- "/share/gtk-doc"
+- "/share/man"
+- "/share/pkgconfig"
+- "*.la"
+- "*.a"
+
+modules:
+- name: talloc
+  buildsystem: autotools
+  sources:
+    - type: archive
+      url: https://www.samba.org/ftp/talloc/talloc-2.3.4.tar.gz
+      sha256: 179f9ebe265e67e4ab2c26cad2b7de4b6a77c6c212f966903382869f06be6505
+
+- name: proot
+  buildsystem: simple
+  build-commands:
+    - mv uthash/src/* src
+    - make -C src proot
+    - install -m755 -d /app/bin
+    - install -m755 -d /app/lib
+    - install -m755 src/proot /app/bin
+  sources:
+    - type: archive
+      url: https://github.com/troydhanson/uthash/archive/refs/tags/v2.3.0.tar.gz
+      sha256: e10382ab75518bad8319eb922ad04f907cb20cccb451a3aa980c9d005e661acc
+      dest: uthash
+    - type: git
+      url: https://github.com/termux/proot
+      commit: 5c462a6ecfddd629b1439f38fbb61216d6fcb359
+
+- name: libadwaita  # needed for AdwEntryRow
+  buildsystem: meson
+  config-opts:
+  - -Dexamples=false
+  - -Dtests=false
+  sources:
+  - type: git
+    url: https://gitlab.gnome.org/GNOME/libadwaita.git
+    tag: 1.2.alpha
+    commit: 9150f4016a04064a00733b78391c8e6aad49bee8
+  modules:
+  - name: libsass
+    buildsystem: meson
+    sources:
+    - type: git
+      url: https://github.com/lazka/libsass.git
+      branch: meson
+      commit: 302397c0c8ae2d7ab02f45ea461c2c3d768f248e
+  - name: sassc
+    buildsystem: meson
+    sources:
+    - type: git
+      url: https://github.com/lazka/sassc.git
+      branch: meson
+      commit: 82803377c33247265d779af034eceb5949e78354
+
+  - name: vte  # needed by gtk4
+    buildsystem: meson
+    config-opts:
+      - -Ddocs=false
+      - -Dvapi=false
+      - -Dgtk4=true
+      - -Dgtk3=false
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/vte.git
+        commit: 3ed53bfaa7344ce5b7727a60a2d33315b15201e5
+
+# - name: fakeroot
+#   buildsystem: simple
+#   build-commands:
+#     - ./configure --prefix=/app
+#     - make
+#     - make install
+#   sources:
+#     - type: archive
+#       url: https://deb.debian.org/debian/pool/main/f/fakeroot/fakeroot_1.29.orig.tar.gz
+#       sha256: 8fbbafb780c9173e3ace4a04afbc1d900f337f3216883939f5c7db3431be7c20
+
+- name: servicectl
+  buildsystem: simple
+  build-commands:
+    - install -m755 serviced /app/bin
+    - install -m755 servicectl /app/bin
+  sources:
+    - type: git
+      url: https://github.com/AtomsDevs/servicectl.git
+      commit: fda22d97a65bb881d83f4ba10a78ce54867eeed4
+
+- name: atoms-core
+  buildsystem: simple
+  build-commands:
+    - python3 setup.py install --prefix=/app --root=/
+  sources:
+    - type: git
+      url: https://github.com/AtomsDevs/atoms-core.git
+      commit: 7fbd7c193bbc0bcb76458d1a5370e1bc20e80e07
+  modules:
```

Comment:
you should use flatpak-builder-tools for the python dependencies.

Reply:
I'll take a look at it. Is this mandatory?

Reply:
it's better for maintenance.

Reply:
I have to do the same for another app that I have already published (Bottles), I would like to do it after publishing if possible